admin/src/pages/UserSearch.vue

```diff
@@ -3,11 +3,23 @@
     <div class="user-search-first-div">
       <b-button class="unconfirmedRegisterMails" variant="light" @click="unconfirmedRegisterMails">
         <b-icon icon="envelope" variant="danger"></b-icon>
-        {{ filterCheckedEmails ? $t('unregistered_emails') : $t('all_emails') }}
+        {{
+          filterByActivated === null
+            ? $t('all_emails')
+            : filterByActivated === false
```

Comment:
This would get wrong, because we switsch between `null` and `false`.

I like that we can see explizit that we switsch between `null` and `false`.
And then the logic works already if we include `true` in the future as well.